### PR TITLE
Remove zignatures and move aaF[l] under F[al] and zf[sdc] under F[sdc] 

### LINKED ISSUE #272: Zignatures search during aaa gives wrong result
Zignature search was tested on a binary builded from this source:
```c
#include <stdio.h>

int main() {
	char buf[5];
	FILE *f = fopen ("test", "r");
	fseek (f, 5, 1);
	fread (buf, 5, 1, f);
	fclose (f);
}
```
This is how `main` looks in the related object file. Ideally I want to see almost the same result after providing `rizin` appropriate zignatures of functions `fopen`, `fseek`, `fread` and `fclose` from `stdio` lib. 
```asm
[0x00000000]> pdf @ sym.main
            ;-- section..text_mn_4:
            ;-- .text$mn:
            ;-- $LN4:
/ 111: int sym.main (int argc, char **argv, char **envp);
|           ; var int64_t var_20h @ rsp+0x20
|           ; var int64_t var_28h @ rsp+0x28
|           ; arg int argc @ rdi
|           ; arg char **argv @ rsi
|           0x00000130      push rbx                                   ; [04] -r-x section size 111 named .text_mn_4
|           0x00000132      sub rsp, 0x30
|           0x00000136      mov rax, qword [sym.imp.__security_cookie] ; [0xfc0:8]=0
|           0x0000013d      xor rax, rsp
|           0x00000140      mov qword [var_28h], rax
|           0x00000145      lea rdx, sym.___C__01KDCPPGHE_r            ; 0xdf0 ; "r"; RELOC 32 ??_C@_01KDCPPGHE@r@ @ 0x00000df0
|           0x0000014c      lea rcx, sym.___C__04CEJDCDCH_test         ; 0xe00 ; "test"; RELOC 32 ??_C@_04CEJDCDCH@test@ @ 0x00000e00
|           0x00000153      call fopen
|           0x00000158      mov edx, 5
|           0x0000015d      mov rcx, rax
|           0x00000160      mov rbx, rax
|           0x00000163      lea r8d, [rdx - 4]
|           0x00000167      call fseek
|           0x0000016c      mov edx, 5
|           0x00000171      lea rcx, [var_20h]
|           0x00000176      mov r9, rbx
|           0x00000179      lea r8d, [rdx - 4]
|           0x0000017d      call fread
|           0x00000182      mov rcx, rbx
|           0x00000185      call fclose
|           0x0000018a      xor eax, eax
|           0x0000018c      mov rcx, qword [var_28h]
|           0x00000191      xor rcx, rsp
|           0x00000194      call __security_check_cookie
|           0x00000199      add rsp, 0x30
|           0x0000019d      pop rbx
\           0x0000019e      ret
```

`fopen` and `fseek` are just small wrappers around internal functions
```asm
/ 11: file*sym.fopen (const char *filename, const char *mode);
|       :   0x000236b0      mov r8d, 0x40                              ; sym..debug_S ; [31] -r-x section size 11 named .text_mn_31
\       `=< 0x000236b6      jmp sym.___common_fsopen_D__YAPEAU_iobuf__QEBD0H_Z; RELOC 32 ??$common_fsopen@D@@YAPEAU_iobuf@@QEBD0H@Z @ 0x00021d70
```

```asm
/ 8: int sym.fseek (FILE *stream, long offset, int whence);
| bp: 0 (vars 0, args 0)
| sp: 0 (vars 0, args 0)
| rg: 1 (vars 0, args 1)
|       :   0x00024680      movsxd rdx, edx                            ; whence ; [46] -r-x section size 8 named .text_mn_46
\       `=< 0x00024683      jmp sym._common_fseek__YAHV__crt_stdio_stream___JH_Z; RELOC 32 ?common_fseek@@YAHV__crt_stdio_stream@@_JH@Z @ 0x00022600
```

`fread` and `fclose` are big enough
```asm
/ 98: int sym.fclose (FILE *stream);
|           ; arg int64_t arg_8h @ rsp+0x30
|           ; arg int64_t arg_10h @ rsp+0x38
|           ; arg FILE *stream @ rdi
|           ; arg int64_t arg4 @ rcx
|           0x00022350      mov qword [arg_10h], rbx                   ; [23] -r-x section size 98 named .text_mn_23
|           0x00022355      mov qword [arg_8h], rcx                    ; arg4
|           0x0002235a      push rdi                                   ; stream
|           0x0002235b      sub rsp, 0x20
|           0x0002235f      mov rbx, rcx                               ; arg4
|           0x00022362      test rcx, rcx                              ; arg4
|       ,=< 0x00022365      jne 0x22385
|       |   0x00022367      call _errno
|       |   0x0002236c      mov dword [rax], 0x16                      ; [0x16:4]=-1
|       |   0x00022372      call _invalid_parameter_noinfo
|       |   ; CODE XREF from sym.fclose @ 0x22394
|      .--> 0x00022377      or eax, 0xffffffff                         ; -1
|      :|   ; CODE XREF from sym.fclose @ 0x223b0
|     .---> 0x0002237a      mov rbx, qword [arg_10h]
|     ::|   0x0002237f      add rsp, 0x20
|     ::|   0x00022383      pop rdi
|     ::|   0x00022384      ret
|     ::`-> 0x00022385      mov eax, dword [rcx + 0x14]                ; arg4
|     ::    0x00022388      shr eax, 0xc
|     ::    0x0002238b      test al, 1
|     ::,=< 0x0002238d      je 0x22396
|     ::|   0x0002238f      call ?__acrt_stdio_free_stream@@YAXV__crt_stdio_stream@@@Z
|     :`==< 0x00022394      jmp 0x22377
|     : `-> 0x00022396      call _lock_file
|     :     0x0002239b      nop
|     :     0x0002239c      mov rcx, rbx
|     :     0x0002239f      call sym._fclose_nolock                    ; RELOC 32 _fclose_nolock @ 0x00022070
|     :     0x000223a4      mov edi, eax
|     :     ;-- $LN10:
|     :     0x000223a6      mov rcx, rbx
|     :     0x000223a9      call _unlock_file
|     :     0x000223ae      mov eax, edi
\     `===< 0x000223b0      jmp 0x2237a
```

```asm
/ 29: size_t sym.fread (void *ptr, size_t size, size_t nmemb, FILE *stream);
|           ; var int64_t var_20h @ rsp+0x20
|           ; arg size_t nmemb @ rdx
|           ; arg int64_t arg5 @ r8
|           ; arg int64_t arg6 @ r9
|           0x00023200      sub rsp, 0x38                              ; [29] -r-x section size 29 named .text_mn_29
|           0x00023204      mov qword [var_20h], r9                    ; arg6
|           0x00023209      mov r9, r8                                 ; arg5
|           0x0002320c      mov r8, rdx                                ; nmemb
|           0x0002320f      or rdx, sym._comp.id_0x01056665            ; nmemb
|           0x00023213      call sym.fread_s                           ; RELOC 32 fread_s @ 0x000233c0
|           0x00023218      add rsp, 0x38
\           0x0002321c      ret
```

Here you can see `main` code after `zo` + `aaa`:
```asm
[0x140001394]> pdf @ main
            ; CALL XREF from sym.__acrt_uninitialize_tmpfile @ 0x14000131f
/ 111: int main (int argc, char **argv, char **envp);
|           ; var int64_t var_20h @ rsp+0x20
|           ; var int64_t var_28h @ rsp+0x28
|           0x140001010      push rbx
|           0x140001012      sub rsp, 0x30
|           0x140001016      mov rax, qword [0x140019008]              ; [0x140019008:8]=0x2b992ddfa232 ; "2\xa2\xdf-\x99+"
|           0x14000101d      xor rax, rsp
|           0x140001020      mov qword [var_28h], rax
|           0x140001025      lea rdx, [0x1400102d0]                    ; "r" ; uint32_t arg2
|           0x14000102c      lea rcx, str.test                         ; 0x1400102d4 ; "test" ; uint32_t arg1
|           0x140001033      call sym.___common_fsopen_D__YAPEAU_iobuf__QEBD0H_Z
|           0x140001038      mov edx, 5                                ; int64_t arg2
|           0x14000103d      mov rcx, rax                              ; int64_t arg1
|           0x140001040      mov rbx, rax
|           0x140001043      lea r8d, [rdx - 4]
|           0x140001047      call sym._common_fseek__YAHV__crt_stdio_stream___JH_Z
|           0x14000104c      mov edx, 5                                ; int64_t arg2
|           0x140001051      lea rcx, [var_20h]                        ; int64_t arg1
|           0x140001056      mov r9, rbx                               ; int64_t arg4
|           0x140001059      lea r8d, [rdx - 4]
|           0x14000105d      call sym._fread_nolock
|           0x140001062      mov rcx, rbx                              ; int64_t arg1
|           0x140001065      call sym._validate_and_update_state_at_beginning_of_format_character___positional_parameter_base_DV__string_output_adapter_D___crt_std
|           0x14000106a      xor eax, eax
|           0x14000106c      mov rcx, qword [var_28h]
|           0x140001071      xor rcx, rsp                              ; uint32_t arg1
|           0x140001074      call fcn.140001110
|           0x140001079      add rsp, 0x30
|           0x14000107d      pop rbx
\           0x14000107e      ret
```
What is wrong:
1. `fopen` and `fseek` got names of internal functions.  IMO it is not ok. Well I understand that these function are too small so simple byte search by mask gives many false positives but search algo should consider xrefs. And also it should detect internal function themselfs that is impossible for now (see https://github.com/radareorg/radare2/issues/16979)
2. `fread` got name of another function `_fread_nolock` which has the same zign bytes (see below). So xrefs issue again.
3. Not sure about `fclose` but seems that is related to xref too
4. Original mangled names are missing since rizin uses filtered unmagled names during zigns generating.

```asm
/ 29: sym._fread_nolock (int64_t arg3, int64_t arg5, int64_t arg6);
|           ; var int64_t var_20h @ rsp+0x20
|           ; arg int64_t arg3 @ rdx
|           ; arg int64_t arg5 @ r8
|           ; arg int64_t arg6 @ r9
|           0x00022400      sub rsp, 0x38                              ; [25] -r-x section size 29 named .text_mn_25
|           0x00022404      mov qword [var_20h], r9                    ; arg6
|           0x00022409      mov r9, r8                                 ; arg5
|           0x0002240c      mov r8, rdx                                ; arg3
|           0x0002240f      or rdx, sym._comp.id_0x01056665            ; arg3
|           0x00022413      call sym._fread_nolock_s                   ; RELOC 32 _fread_nolock_s @ 0x000225c0
|           0x00022418      add rsp, 0x38
\           0x0002241c      ret
```

Also check `pdf` output of `main` after `zo` + `z/` + `aa;aac;aar`:
```asm
[0x140001394]> pdf @ main
            ; CALL XREF from entry0 @ 0x14000131f
/ 111: int main (int argc, char **argv, char **envp);
|           ; var int64_t var_20h @ rsp+0x20
|           ; var int64_t var_28h @ rsp+0x28
|           0x140001010      push rbx
|           0x140001012      sub rsp, 0x30
|           0x140001016      mov rax, qword [0x140019008]              ; [0x140019008:8]=0x2b992ddfa232 ; "2\xa2\xdf-\x99+"
|           0x14000101d      xor rax, rsp
|           0x140001020      mov qword [var_28h], rax
|           0x140001025      lea rdx, [0x1400102d0]                    ; "r"
|           0x14000102c      lea rcx, str.test                         ; 0x1400102d4 ; "test"
|           0x140001033      call fcn.1400033cc
|           0x140001038      mov edx, 5
|           0x14000103d      mov rcx, rax
|           0x140001040      mov rbx, rax
|           0x140001043      lea r8d, [rdx - 4]
|           0x140001047      call fcn.140003938
|           0x14000104c      mov edx, 5
|           0x140001051      lea rcx, [var_20h]
|           0x140001056      mov r9, rbx
|           0x140001059      lea r8d, [rdx - 4]
|           0x14000105d      call sign.bytes.sym.fread_0
|           0x140001062      mov rcx, rbx
|           0x140001065      call sign.bytes.sym.fclose_0
|           0x14000106a      xor eax, eax
|           0x14000106c      mov rcx, qword [var_28h]
|           0x140001071      xor rcx, rsp
|           0x140001074      call fcn.140001110
|           0x140001079      add rsp, 0x30
|           0x14000107d      pop rbx
\           0x14000107e      ret
```
Ok, `fopen` and `fseek` are not detected but `fread` and `fclose` are ok (except for this `_0` suffix in their names).

**Drag and drop zip archives containing the Additional info here, don't use external services or link.**
Binary + object file + stdio zigns: [zign_workdir.zip](https://github.com/rizinorg/rizin/files/5744453/zign_workdir.zip)

Signatures are implemented in:
- `librz/core/cmd_zign.c`
- `librz/analysis/sign.c`

### LINKED ISSUE #973: Remove legacy signatures ("zignatures") support
Current format of signatures serialization uses SDB and separates values with commas (`,`) which is often problematic. Moreover, this storage format is inefficient.

We need to reimplement the storing and loading, consider some denser format as well.

See the current implementation in:
- `librz/core/cmd_zign.c`
- `librz/analysis/sign.c`

### LINKED ISSUE #1300: Fix type matching for signatures
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | -
| File format of the file you reverse (mandatory)      | ELF
| Architecture/bits of the file (mandatory)            | x86/64
| `rizin -v` full output, **not truncated** (mandatory)  | 

### Expected behavior

Successful rename of the function arguments

### Actual behavior

```
[XX] db/cmd/cmd_zignature add types and function name on sig match
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'zs *
za main o 0x00004070
za main t func.sym.new_function_name.ret="int" func.sym.new_function_name.args=2 func.sym.new_function_name.arg.0="\"int,NEWARGC\"" func.sym.new_function_name.arg.1="\"char **,NEWARGV\""
za main n sym.new_function_name
s main
af
e zign.match.offset=true
z.
afs
afn
' bins/elf/ls
-- stdout
--- expected
+++ actual
@@ -1,2 +1,2 @@
-int sym.new_function_name(int NEWARGC, char ** NEWARGV);
+void sym.new_function_name(int argc, char ** argv);
 sym.new_function_name
```

### LINKED ISSUE #1311: Fix or improve non-C type tests
Fix broken type tests:
```
[XX] db/cmd/cmd_zignature add types and function name on sig match
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'zs *
za main o 0x00004070
za main t func.sym.new_function_name.ret="int" func.sym.new_function_name.args=2 func.sym.new_function_name.arg.0="\"int,NEWARGC\"" func.sym.new_function_name.arg.1="\"char **,NEWARGV\""
za main n sym.new_function_name
s main
af
e zign.match.offset=true
z.
afs
afn
' bins/elf/ls
-- stdout
--- expected
+++ actual
@@ -1,2 +1,2 @@
-int sym.new_function_name(int NEWARGC, char ** NEWARGV);
+void sym.new_function_name(int argc, char ** argv);
 sym.new_function_name
-- stderr
[+] searching 0x00004070 - 0x00004170
Cannot find bytes for this signature: main
[+] searching function metrics
hits: 1

[XX] db/cmd/dwarf "function info integration ada"
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e asm.bytes=true
e asm.dwarf=false
aaa
pdf @  dbg.main
' bins/elf/ada_test_dwarf
-- stdout
--- expected
+++ actual
@@ -5,7 +5,7 @@
 |           ; var char **var_20h @ rbp-0x20
 |           ; var int var_14h @ rbp-0x14
 |           ; var system__address ensure_reference @ rbp-0x10
-|           ; var ada_main__main__seh___PAD seh @ rbp-0x8
+|           ; var int64_t var_8h @ rbp-0x8
 |           ; arg int argc @ rdi
 |           ; arg char **argv @ rsi
 |           ; arg char **envp @ rdx
@@ -23,7 +23,7 @@
 |           0x0000276d      488905b42900.  mov   qword [obj.gnat_argv], rax ; [0x5128:8]=0
 |           0x00002774      488b45d8       mov   rax, qword [var_28h]
 |           0x00002778      488905112900.  mov   qword [obj.gnat_envp], rax ; [0x5090:8]=0
-|           0x0000277f      488d45f8       lea   rax, [seh.F]
+|           0x0000277f      488d45f8       lea   rax, [var_8h]
 |           0x00002783      4889c7         mov   rdi, rax
 |           0x00002786      e875f9ffff     call  sym.imp.__gnat_initialize
 |           0x0000278b      e8e8fcffff     call  dbg.adainit
-- stderr
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function calls (aac)
[x] Analyze len bytes of instructions for references (aar)
[x] Check for classes
[x] Type matching analysis for all functions (aaft)
[x] Propagate noreturn information
[x] Integrate dwarf function information.
[x] Use -AA or aaaa to perform additional experimental analysis.

[XX] db/cmd/dwarf "big endian variable info"
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e asm.bytes=true
e asm.dwarf=false
aaa
pdf @  dbg.init
' bins/elf/ppc64_sudoku_dwarf
-- stdout
--- expected
+++ actual
@@ -1,8 +1,8 @@
             ; CALL XREF from dbg.main @ 0x10001ae8
             ;-- Grid::init():
             ;-- method.Grid.init:
-/ dbg.init (Grid * const this);
-|           ; arg Grid * const this @ r3
+/ dbg.init (int64_t arg1);
+|           ; arg int64_t arg1 @ r3
 |           0x100012bc      39030144       addi  r8, r3, 0x144         ; arg1 ; void init(Grid * const this);
 |           0x100012c0      38e30168       addi  r7, r3, 0x168         ; arg1
 |           0x100012c4      39400000       li    r10, 0
-- stderr
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function calls (aac)
[ ] Analyze len by;22mint64_t var_20h @ rbp-0x20
+|           ; var int64_t var_18h @ rbp-0x18
 |           0x0000123b      f30f1efa       endbr64                     ; int main();
 |           0x0000123f      55             push  rbp
 |           0x00001240      4889e5         mov   rbp, rsp
@@ -12,8 +12,8 @@
 |           0x00001248      bf08000000     mov   edi, 8
 |           0x0000124d      e81efeffff     call  sym.imp.operator_new_unsigned_long
 |           0x00001252      4889c3         mov   rbx, rax
-|           0x00001255      4889df         mov   rdi, rbx
+|           0x00001255      4889df         mov   rdi, rbx              ; int64_t arg1
 |           0x00001258      e827010000     call  method Bird::Bird()   ; dbg.Bird::Bird()
-|           0x0000125d      48895de0       mov   qword [b], rbx
+|           0x0000125d      48895de0       mov   qword [var_20h], rbx
 |           0x00001261      bf08000000     mov   edi, 8
 |           0x00001266      e805feffff     call  sym.imp.operator_new_unsigned_long
-- stderr
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function calls (aac)
[x] Analyze len bytes of instructions for references (aar)
[x] Check for classes
[x] Type matching analysis for all functions (aaft)
[x] Propagate noreturn information
[x] Integrate dwarf function information.
[x] Use -AA or aaaa to perform additional experimental analysis.
```

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

 command changed:
- `zfc` to `Fc <outfile>` FLIRT create signature file
- `zfd` to `Fd <infile>` FLIRT dump signature file
- `zfs` to `Fs <infile>` FLIRT apply signature file
- `aaF` to `Fa <filter>` SigDB apply signatures
- `aaFl` to `Fl` SigDB list signatures

Fix #272
Fix #973
Fix #1311
Fix #1300

### CODE REVIEW

File: librz/include/rz_flirt.h [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
+#include <rz_types.h>
 #include <rz_list.h>
 #include <rz_analysis.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+RZ_LIB_VERSION_HEADER(rz_sign);
```
Comment (ret2libc) [inline librz/include/rz_flirt.h]: rz_flirt ? shall we change also the name of the lib?

**Comment by ret2libc** [comment]
So we want to just kill zignatures completely?

**Comment by wargio** [comment]
> So we want to just kill zignatures completely?

yes.

File: librz/signature/sign.c [hunk 2]
```diff
@@ -2,6 +2,6 @@
 // SPDX-FileCopyrightText: 2021 deroad <wargio@libero.it>
 // SPDX-License-Identifier: LGPL-3.0-only
 
-#include <rz_sign.h>
+#include <rz_flirt.h>
 
-RZ_LIB_VERSION(rz_sign);
+RZ_LIB_VERSION(rz_flirt);
```
Comment (XVilka) [inline librz/signature/sign.c]: I do not agree with this change - while the underlying technology of our signatures is FLIRT, the API and module name should remain generic. FLIRT is a relatively old technology, maybe at some point, we will decide to implement something newer and better. Thus, the name of the module and most of the API should remain the same - `rz_sign`.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/signature/sign.c (hunk 2 above)
Comment (ret2libc) [reply librz/signature/sign.c]: TBH at this point the api does not look too generic. see https://github.com/rizinorg/rizin/blob/dfdc45156550f4532beac89ad2319419ab14dd6c/librz/include/rz_flirt.h, RzFlirtCompressedOptions, rz_sign_flirt_parse_header_compressed_pattern_from_buffer, rz_sign_flirt_parse_string_pattern_from_buffer, rz_sign_flirt_apply, rz_sign_flirt_node_count_nodes, RzFlirtInfo, etc. etc.

Comment (wargio) [reply librz/signature/sign.c]: i'm ok with both options.

Comment (wargio) [reply librz/signature/sign.c]: ok, i have decided to revert it back to `rz_sign` since the lib is called `rz_sign`.

Comment (XVilka) [reply librz/signature/sign.c]: Thank you.

**Review by XVilka — APPROVED** [review]

**Review by ret2libc — CHANGES_REQUESTED** [review]
I'm ok with everything in the PR, apart that if there is a `rz_sign` library there should probably be a `rz_sign.h` header.

**Comment by wargio** [comment]
makes sense, i will revert it back that change.

**Review by XVilka — APPROVED** [review]

File: librz/include/rz_sign.h [hunk 3]
```diff
@@ -2,165 +2,14 @@
 #define RZ_SIGN_H
```
Comment (ret2libc) [inline librz/include/rz_sign.h:2]: this file is almost empty now. is it used only to include rz_flirt.h?

Comment (wargio) [reply librz/include/rz_sign.h:2]: yes, and the header for the version.

Comment (wargio) [reply librz/include/rz_sign.h:2]: i reverted it back as you have asked.

**Comment by ret2libc** [comment]
I don't like too much having `aaF` and `aaFl` separated from `F`. I think all the FLIRT management stuff should be in one place. WDYT?

**Comment by wargio** [comment]
> I don't like too much having `aaF` and `aaFl` separated from `F`. I think all the FLIRT management stuff should be in one place. WDYT?

i was actually thinking the same, probably is better to rename everything like:
- `Ffc <outfile>` FLIRT create signature file
- `Ffd <infile>` FLIRT dump signature file
- `Ffs <infile>` FLIRT apply signature file
- `Fds <filter>` SigDB apply signatures
- `Fdl` SigDB list signatures

File: librz/core/cmd_descs/cmd_flirt.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_flirt
+commands:
+  - name: Ffc
```
Comment (ret2libc) [inline librz/core/cmd_descs/cmd_flirt.yaml]: what about `Fc`?

File: librz/core/cmd_descs/cmd_flirt.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    cname: flirt_create
+    summary: Create a FLIRT file (.pat or .sig)
+    args:
+      - name: filename
+        type: RZ_CMD_ARG_TYPE_FILE
+  - name: Ffd
```
Comment (ret2libc) [inline librz/core/cmd_descs/cmd_flirt.yaml]: `Fd`?

File: librz/core/cmd_descs/cmd_flirt.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    cname: flirt_dump
+    summary: Open a FLIRT file (.pat or .sig) and dumps its contents
+    args:
+      - name: filename
+        type: RZ_CMD_ARG_TYPE_FILE
+  - name: Ffs
```
Comment (ret2libc) [inline librz/core/cmd_descs/cmd_flirt.yaml]: `Fs`

File: librz/core/cmd_descs/cmd_flirt.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    cname: flirt_scan
+    summary: Open a FLIRT file (.pat or .sig) and tries to apply the signatures to the loaded binary
+    args:
+      - name: filename
+        type: RZ_CMD_ARG_TYPE_FILE
+  - name: Fds
```
Comment (ret2libc) [inline librz/core/cmd_descs/cmd_flirt.yaml]: `Fa` (Flirt apply)

File: librz/core/cmd_descs/cmd_flirt.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    summary: Apply signatures from sigdb
+    cname: apply_signatures_from_sigdb
+    args:
+      - name: filter
+        type: RZ_CMD_ARG_TYPE_STRING
+        optional: true
+  - name: Fdl
```
Comment (ret2libc) [inline librz/core/cmd_descs/cmd_flirt.yaml]: `Fl`?

**Review by XVilka — APPROVED** [review]

**Review by ret2libc — COMMENTED** [review]
Can we rename the folder to `librz/sign`? Also, do I remember wrong or we agreed to keep the rz_sign.h header for consistency with the name of the rz-module?

**Comment by wargio** [comment]
> Can we rename the folder to `librz/sign`? Also, do I remember wrong or we agreed to keep the rz_sign.h header for consistency with the name of the rz-module?

Yes for rz_sign and and is in there.

**Review by ret2libc — CHANGES_REQUESTED** [review]
Please rename librz/signature to librz/sign, apart from that this LGTM.

**Review by XVilka — APPROVED** [review]

**Pull request merged**